packages/table/src/Table.js

```diff
@@ -529,39 +529,44 @@ const headers = tableConfig => tableConfig.fields
   .filter(field => !field.disappear)
   .map(field => getHeaderCell(field))
 
-const isRowLoaded = (tableData, loadLookAhead) => ({ index }) => {
-  if (Array.isArray(tableData)) return !!tableData[index + loadLookAhead]
-  else if (Immutable.List.isList(tableData)) {
-    // Console.log(index, loadLookAhead, tableData.get(index + loadLookAhead))
-    return tableData.get(index + loadLookAhead)
+
+function getRowData(tableData, rowIndex) {
+  if (Array.isArray(tableData)) {
+    return tableData[rowIndex]
+  }
+
+  if (Immutable.List.isList(tableData)) {
+    return tableData.get(rowIndex)
   }
-  return false
+
+  return undefined
 }
 
-const tableRowRenderer = (tableConfig, tableData, searchText, showIndex, onRowClick, onRowHover) =>
-  ({ key, index, style }) => {
-    let tData
-    if (Array.isArray(tableData)) tData = tableData[index]
-    else if (Immutable.List.isList(tableData)) {
-      tData = tableData.get(index)
-    }
-    if (!isRowLoaded(tableData, 25)) {
-      return <div>Loading</div>
-    }
-    const row = getDataRow(tableConfig, tData, searchText, index, showIndex, onRowClick, onRowHover)
+
+function tableRowRenderer (tableConfig, tableData, searchText, showIndex, onRowClick, onRowHover) {
+  return function({ key, index, style }) {
```

Comment:
A stylistic change, but I find `return function() { ... }` easier to read than arrow functions returning arrow functions.